### PR TITLE
Oa/icons script and update icons

### PR DESCRIPTION
**Pull request by oliverabrahams**
## What are you changing?
Add the 'bin' icon.
Update 'reload' icon.
Add 'notification-off-round' icon.
Add 'notification-on-round' icon.

- Updating the icons https://github.com/guardian/design-system-advocates/issues/4 
- Updated the script to use a different Figma page for icons 
- Added an **_await_** to the format function from prettier

## Why?

The figma files have changed location so the script needed updating. 

https://github.com/guardian/design-system-advocates/issues/4

### CODE REVIEW

**Comment by mxdvl** [comment]
May I ask if this could include the [round notifications icons](https://www.figma.com/file/b2qv2OMLoNCYnP01ipfrP7/%E2%97%88-Core-library?type=design&node-id=4654-4505&mode=design&t=M38r3o6TzECGTAhS-4) which we want to use in [the new Article Meta](https://github.com/guardian/dotcom-rendering/pull/9409#discussion_r1383740188)

image

**Comment by oliverabrahams** [comment]
> May I ask if this could include the [round notifications icons](https://www.figma.com/file/b2qv2OMLoNCYnP01ipfrP7/%E2%97%88-Core-library?type=design&node-id=4654-4505&mode=design&t=M38r3o6TzECGTAhS-4) which we want to use in [the new Article Meta](https://github.com/guardian/dotcom-rendering/pull/9409#discussion_r1383740188)

Of course, no problem

File: .changeset/good-pants-sin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components': major
+---
+
+Added icon bin, notification on round, notification off round, updated share callout and reload icons
```
Comment (sndrs) [inline .changeset/good-pants-sin.md:5]:
```suggestion
## New icons
- `SvgBin`
- `SvgNotificationsOffRound`
- `SvgNotificationsONRound`

## Updated icons
- `SvgShareCallout` 
- `SvgReload`
```

**Review by sndrs — COMMENTED** [review]
remember this is what goes in the changelog, the more we can help people know what/if they need to check the easier it is for them to updrade

**Review by sndrs — APPROVED** [review]

**Pull request merged**